docs/dev/agent_integration.md

```diff
@@ -73,7 +138,7 @@ The `ddev create` command runs an interactive tool that creates the basic file a
    ddev create -n Awesome
    ```
 
-   This command displays the path where the files would have been written, as well as the structure itself. Make sure the path in the first line of output matches your `integrations-extras` repository location.
+   This command displays the path where the files would have been written, as well as the structure itself. Make sure the path in the first line of output matches your `integrations-extras` or `marketplace` repository location.
 
 1. Run the command without the `-n` flag. The tool asks you for an email and name and then creates the files you need to get started with an integration.
```

Comment:
Do we have any guidance on how partners should name their integrations? For example, in the Marketplace, we need users to follow a {partner}.{integration technology} convention, because more than one partner could create an integration for the same technology. 

Also, if the user runs the command with the integration name "awesome" in the example, but then decides to use that as a template to create their own check, where will they need to change out the name "awesome"? Is it only picked up by the name of a directory, or is it added in other places? I just want to ensure that if all users start by  creating "awesome" checks, that it doesn't causes issues when they go to open a pull-request for their customer integration.